src/PokemonTrainer.java

```diff
@@ -1,6 +1,149 @@
-import java.util.List;
+import java.util.*;
 
 public class PokemonTrainer {
     private String name;
-    List<Pokemon> pokemons;
+    List<Pokemon> benchPokemons = new ArrayList<>();
+    Pokemon attackPokemon;
+    List<String> cardDeckEnergieAndFood = new ArrayList<>();
+
+    public PokemonTrainer(String name) {
+        this.name = name;
+    }
+
+    public void chooseAttack(PokemonGymImpl gym) {
+        int placeholderAnswer;
+
+        attackPokemon.printAttacks();
+        placeholderAnswer = PokemonGymImpl.askingInput("Which attack do you choose? Type in the number of the attack.");
+        attackPokemon.doAttack(placeholderAnswer, gym);
+    }
+
+    public void choosePokemon(Pokemon enemy) {
+        this.printBenchPokemon();
+        int placeholderAnswer = PokemonGymImpl.askingInput("Which do you choose from your deck? Type in the number of the pokemon.");
+        this.attackPokemon = benchPokemons.get(placeholderAnswer);
+        benchPokemons.remove(placeholderAnswer);
+        System.out.println(attackPokemon.getName() + " is ready to fight his opponent");
+        attackPokemon.makeSound();
+    }
+
+    public void printBenchPokemon() {
```

Comment:
Is het nodig om te differentieren tussen attackpokemon en benched pokemons? Als je een aanval doet, doe je dan niet "pikachu.doAttack()" ? Dan weet je toch dat het pikachu is die de attack doet?